src/components/Top10.js

```diff
@@ -0,0 +1,108 @@
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import amzn from '../assets/amzn.png';
+import aapl from '../assets/apple.webp';
+import msft from '../assets/msft.png';
+import google from '../assets/google.png';
+import TSLA from '../assets/tsla.png';
+import NVDA from '../assets/nvda.png';
+import UNH from '../assets/UnitedHealth-Group-Logo-1977.png';
+import META from '../assets/meta.png';
+import BRK from '../assets/brk.png';
+import Visa from '../assets/visa.png';
+import { getView, makeView } from '../redux/features/stock/stockSlice';
+
+const Top10 = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getView());
+  }, []);
+
+  const { Views } = useSelector((state) => state.stock);
+  if (Views.length < 1) {
+    return (
+      <h1 className="loading">Loading ...</h1>
+    );
+  }
+  return (
+    <div>
+      <h1 className="title">Top10</h1>
+      <div className="links">
+        <Link to="/AAPL" className="stock" onClick={() => dispatch(makeView('AAPL'))}>
+          <p className="views">
```

Comment:
*  According to the requirements, this being the home page,the data filtration should be done here [check requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=When%20the%20page,the%20details%20page.)